src/component/contents/__tests__/DraftEditorBlock.react-test.js

```diff
@@ -249,7 +250,7 @@ describe('DraftEditorBlock.react', () => {
       );
       var decorator = new Decorator();
 
-      var newTree = BlockTree.generate(helloBlock, decorator);
+      var newTree = BlockTree.generate(ContentState.createFromText(helloBlock.getText()), helloBlock, decorator);
```

Comment:
Internally we have an 80-col limit, so even if lint doesn't complain here we'll need to end up cleaning it up later.